src/analysis/normed_space/dual.lean

```diff
@@ -96,3 +109,80 @@ end
 end bidual_isometry
 
 end normed_space
+
+namespace inner_product_space
+open is_R_or_C continuous_linear_map
+
+variables (𝕜 : Type*)
+variables {E : Type*} [is_R_or_C 𝕜] [inner_product_space 𝕜 E]
+local notation `⟪`x`, `y`⟫` := @inner 𝕜 E _ x y
+local postfix `†`:90 := @is_R_or_C.conj 𝕜 _
+
+/--
+Given some x in an inner product space, we can define its dual as the continuous linear map
+λ y, ⟪x, y⟫.
+-/
+def to_dual (x : E) : normed_space.dual 𝕜 E :=
+linear_map.mk_continuous
+{ to_fun := λ y, ⟪x, y⟫,
+  map_add' := by simp only [inner_add_right, forall_const, eq_self_iff_true],
+  map_smul' := by simp [inner_smul_right] }
+∥x∥
+(λ y, by { rw [is_R_or_C.norm_eq_abs], exact abs_inner_le_norm _ _ })
+
```

Comment:
Can you establish also that ` λ x, to_dual x` is a continuous linear map, then your Riesz representation theorem can be that that map is a continuous linear equivalence?  (Or even an isometry, if you set up the inner product space structure on the dual of an inner product space.)

Reply:
The problem is that it's not linear, it's conjugate-linear... We should probably have some API for this concept at some point!

Reply:
> The problem is that it's not linear, it's conjugate-linear... We should probably have some API for this concept at some point!

Shall we start on Zulip the discussion of how conjugate-linear maps should work?  I am not sure of the right generality (maybe there are finite extensions of Q which have conjugates and are used in this way).  And probably is_R_or_C.conj_to_ring_equiv 
 gets used at some point.

It may (likely) not be quickly-enough implemented to help with this PR, but it'd be nice to get the process started.

Reply:
Could you please add a `continuous_linear_equiv` version for the case `𝕜 = real`?

Reply:
Looks like this one will take a bit more time, it turns out I need some other results first, like showing that the dual space is complete.

Reply:
This is now done!